src/ast/irbuilderbpf.cpp

```diff
@@ -1783,8 +1783,24 @@ Value *IRBuilderBPF::CreateStrcontains(Value *haystack,
   return CreateIntCast(phi, getInt64Ty(), false);
 }
 
-CallInst *IRBuilderBPF::CreateGetNs(TimestampMode ts, const Location &loc)
+Value *IRBuilderBPF::CreateGetNs(TimestampMode ts,
+                                 const Location &loc,
+                                 const std::string &dummy_ts_map)
 {
+  if (!dummy_ts_map.empty()) {
+    AllocaInst *key = CreateAllocaBPF(getInt64Ty(), "dummy_ts_map_key");
+    CreateStore(getInt64(0), key);
```

Comment:
Maybe add a check for the map key type and size to make sure it fits our assumptions. This is a test fixture, but some validation would still be nice.